cl/opinion_page/templates/view_opinion.html

```diff
@@ -148,6 +148,29 @@ <h3><span>Visualizations</span></h3>
             </div>
         {% endif %}
 
+        {# Recommendations #}
+        {% if recommendations %}
+            <div id="recommendations" class="sidebar-section">
+                <h3><span>Related Opinions</span></h3>
+                <p class="bottom">The following opinions cover similar topics:</p>
+                <ul>
+                    {% for rec in recommendations %}
+                        <li>
+                            <a href="{{ rec.absolute_url }}?{{ request.META.QUERY_STRING }}">
+                                {{ rec.caseName|safe|truncatewords:10|v_wrapper }}
+                            </a>
+                        </li>
+                    {% endfor %}
+                </ul>
+                <p>
+                    <a href="{% url 'show_results' %}?q=related:opinion{{ cluster.pk }}"
```

Comment:
Any reason to say the object type in the query? If users are going to be able to do this, why not make the query:

    related:$(id)

Instead of:

    related:opinion$(id)

?

Reply:
This still comes from my misunderstanding that the search allows mixed-content. Since this is not the case, I'll change it `related:$id`.